openhtf/util/data.py

```diff
@@ -181,9 +181,9 @@ def _sizeof(current_obj):
 
     if isinstance(current_obj, dict):
       size += sum(map(sizeof, itertools.chain.from_iterable(
-          current_obj.iteritems())))
+          iter(current_obj.items()))))
```

Comment:
Unsure what this should look like, since http://python-future.org/compatible_idioms.html#iterating-through-dict-keys-values-items suggests a few strategies. Does this currently have test coverage?

Reply:
This should. We can remove the `iter()` call here since `.items()` is an iterable in PY2 and 3